packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.6.1
```

Comment:
This is adding new public API, so should be 2.7.0 according to semver rules.

Reply:
Renamed to 2.8.0